docs/node-operators/migration-guides/update-network-with-checkpoint.md

```diff
@@ -0,0 +1,265 @@
+
+The binaries you need: 
+* Vega binary: Also includes Tendermint and data node as binary subcommands
+* Visor binary: Optional for setting up Visor for protocol upgrades (See the docs listed in [step 7](#7-read-the-visor-documentation) for information on Visor.
+
+See example commands for downloading below. You may need to update the version number depending on the version of the binaries you need to update to:
+
+```bash
+# Download archives
+wget https://github.com/vegaprotocol/vega/releases/download/v0.68.0/vega-linux-amd64.zip
+wget https://github.com/vegaprotocol/vega/releases/download/v0.68.0/visor-linux-amd64.zip
+
+# Unzip downloaded archives
+unzip vega-linux-amd64.zip
+unzip visor-linux-amd64.zip
+
+mv vega <VEGA-BIN>
+mv visor <VISOR-BIN>
+```
+
+:::node  Manual update for binaries when you are running visor
+Visor cannot automatically restart(or upgrade) binaries because We are restarting the network with the `checkpoint`. Once you stop the Vega network, you must update the old binaries with the downloaded ones. You can find the binary path in the `<VEGAVISOR-HOME>/current/run-config.toml` file. Usually the path is `<VEGAVISOR-HOME>/current/vega`
+:::
+
+### 4. Reset and clear all data
+
+:::warning Back up files before progressing
+Ensure you have a backup of the network files because the steps below will remove data from your home.
+
+You may also risk losing your wallets, so back them up as well.
+:::
+
+1. Call unsafe reset all for Tendermint: `<VEGA-BIN> tendermint unsafe-reset-all --home <TENDERMINT-HOME>`
+2. Call unsafe reset all for vega core: `<VEGA-BIN> unsafe_reset_all --home <VEGA-NETWORK-HOME>`
+3. Remove the data-node state directory: `rm -f <VEGA-NETWORK-HOME>/state/data-node`
+4. Recreate the PostgreSQL database if you have data within: 
+    - a. Call the following command in PostgreSQL terminal: `DROP DATABASE IF EXISTS <VEGA-DB-NAME>`
+    - b. Follow instructions in the step to [Install/Upgrade PostgreSQL instance](./upgrade-node.md#15-installupgrade-postgresql-for-data-node) (optional for data node setup) to recreate new database
+
+
+### 5. Prepare genesis file
+We recommend doing this at the beginning of the upgrade procedure, but this can happen at any point before validators start the network. After the genesis is prepared, all the validators must use the same `genesis.json` file.
+
+To load the checkpoint, find more information in the [restart network guide](../how-to/restart-network.md#load-checkpoint) 
+
+1. One of the Vega team members will adjust [the genesis file ↗](https://github.com/vegaprotocol/networks/blob/master/mainnet1/genesis.json) in the [Vega Protocol networks repository ↗](https://github.com/vegaprotocol/networks).
+2. The person responsible for updating genesis needs to create a PR with changes.

```

Comment:
Ideally, this should be done by validators.
I would say they can ask for vega team member to review it.

Reply:
Rephrased